cmd/storer2elastic/dataprocessor/dataReplayer_test.go

```diff
@@ -263,6 +264,129 @@ func TestDataReplayer_Range_EpochStartMetaBlockFoundInNonceHashStorageShouldWork
 	_ = dr.Range(handlerFunc)
 }
 
+func TestDataReplayer_Range_TransactionInPool(t *testing.T) {
```

Comment:
This test has no asserts. Some other tests with Range have no asserts, is it ok?